lib/oc-tests/test_shareDir.py

```diff
@@ -95,6 +95,13 @@ def setup(step):
     check_users(config.oc_number_test_users)
 
     reset_rundir()
+    reset_server_log_file()
```

Comment:
It should not be necessary to add any hardcoded number (also, remember that these functions [reset_owncloud_account] will be removed from the testcase itself into common setup_test()).

You probably do not need to do anything for existing config files, just the config template that is in use would be set to None rather than 3 so for new config files you'd need to set it yourself.